tests/templates/subroutines/arithmetic/test_phase_adder.py

```diff
@@ -236,10 +236,26 @@ def test_decomposition(self):
         mod = 7
         work_wire = [0]
 
-        phase_adder_decomposition = qml.PhaseAdder(
+        phase_adder_decomp = qml.PhaseAdder(k, x_wires, mod, work_wire).compute_decomposition(
```

Comment:
`compute_decomposition` is a static method, so we don't need to initialize the instance
```suggestion
        phase_adder_decomp = qml.PhaseAdder.compute_decomposition(
```